tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -179,7 +179,7 @@
     #dataTableContainer
     [ngClass]="{
       'data-table-container': true,
-      'expanded': cardState!.tableExpanded
+      'expanded': cardState!.tableExpanded || cardState!.fullSize
```

Comment:
This will prevent users from collapsing the tables of cards that are fullSize. You should update the cardState when the card is set to full size. I left a suggestion that should handle this.
```suggestion
      'expanded': cardState!.tableExpanded
```

Reply:
I feel fullsize mode is a bit misunderstanding and the relationship of design is confusing. In my mind when you're in full size the table is always expanded.
If we want to have two isolated control (and this part is not clear to me), then attribute of CardState shuld be heightExpanded and widthExpanded. With fullsize toggle both and tableExpanded toggle only the height
Then the logic here is much more clear (cardState!.heightExpanded)

Reply:
That said, without `cardState!.fullSize` here the table will not be expanded. Is that what you want.
We can update both fullsize and tableExpanded value in the reducer, but that seems a bit inconsistent from the action (name) to me